mmv1/products/securesourcemanager/Instance.yaml

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Instance'
+kind: 'securesourcemanager#instance'
+base_url: 'projects/{{project}}/locations/{{location}}/instances?instance_id={{instance_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/instances/{{instance_id}}'
+update_verb: :PATCH
```

Comment:
I don't see the PATCH method in the API doc, so the resource should be immutable and has `immutable: true` instead of having the update_verb. I wonder what is the reason to add `update_verb` here.

Reply:
Our update API is currently not used, but I think you're right, I should remove it for now and have it set as immutable. I assume if update is needed, this can be changed fairly easily.

Reply:
It is easy to support update method in the future.